tests/testthat/test-transition_risk_profile.R

```diff
@@ -150,3 +148,63 @@ test_that("outputs columns `transition_risk_category_share` and `transition_risk
 
   expect_true(all(unique(expected_cols) %in% names(company_level_output)))
 })
+
+test_that("outputs `NA` in `avg_transition_risk_best_case` and `avg_transition_risk_worst_case` for `NA` at company level if `transition_risk_score` and `transition_risk_category` are `NA` at product level", {
+  restore <- options(list(
+    readr.show_col_types = FALSE,
```

Comment:
This should be unnecessary since we do it for all tests in tests/testthat/setup.R

https://github.com/2DegreesInvesting/tiltIndicatorAfter/blob/main/tests/testthat/setup.R